docs/internal-documentation/maintenance/ReleaseProcess.md

```diff
@@ -142,49 +166,19 @@ Write a detailed instruction with commands.
 
 ## 12. Tag the release with git
 
-Tag the release with these git commands, and push the tag to Github.
-
-**Change the version number below to the correct version for this release**
-
-zonemaster-ldns:
-
-    git tag v1.0.0
-	git push <repository> --tags
-
-zonemaster-engine:
-
-    git tag v1.0.0
-	git push <repository> --tags
-
-zonemaster-cli:
-
-    git tag v1.0.0
-	git push <repository> --tags
-
-zonemaster-backend:
-
-    git tag v1.0.0
-	git push origin --tags
-
-zonemaster-gui:
-
-    git tag v1.0.0
-	git push origin --tags
-
-### ToDo
-
-Write a description how to set release in Github to get a nice presentation.
+For each repository, go to "releases" in Github and select "draft a new release".
+Use the version number as tag and create a new release description in the same
+format as previous releases. 
```

Comment:
For zonemaster-gui, add the distributed tarballs as attached file of the new release.

Reply:
Fixed.